src/commands/skip.ts

```diff
@@ -31,6 +31,7 @@ class SkipCommand implements BaseCommand {
             await gameSession.endRound();
             startGame(gameSession, guildPreference, db, message, client);
             logger.info(`${getDebugContext(message)} | Skip majority achieved.`);
+            gameSession.lastActive = Date.now();
```

Comment:
Do we only want to consider activity when skip majority is achieved? Or when a skip vote is received too

Reply:
I'm fine either way. What do you prefer?

Reply:
think it should be on both

Reply:
I'll update accordingly